go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/hashicorp/terraform-config-inspect v0.0.0-20231204233900-a34142ec2a72
 	github.com/manifoldco/promptui v0.9.0
 	github.com/mitchellh/go-homedir v1.1.0
+	github.com/op/go-logging v0.0.0-20160315200505-970db520ece7
```

Comment:
Should we be using a library that's 8 years old and hasn't been updated since?

Reply:
Fair point. 
I could have a look into https://pkg.go.dev/go.uber.org/zap and test that. Looks a slight bit more complicated but is up-to-date

Reply:
`slog` and `log` from @yermulnik look pretty good. I'm not a Go dev so happy to take advice. Just don't want to end up with dependencies on a dead project :) 

No objections to `zap` - just seems overly complex (geared towards performance it seems) for what we'd need?

Reply:
I'm all for simplicity (vs complexity). Though I'm not a Go dev too =)
Let's rely upon @MatrixCrawler preference on whichever of these two (Charmbracelet or Gookit) to use.

Reply:
Let's go for the gookit logger then :)